From 511dae4afbfa9ba7a687909405027580f5bc93c5 Mon Sep 17 00:00:00 2001
From: Li Liu <quic_lliu6@quicinc.com>
Date: Thu, 10 Oct 2024 16:12:58 +0800
Subject: [PATCH 2/9] FROMLIST: dt-bindings: display/msm: dsi-controller-main:
 Document QCS615

Document general compatibility of the DSI controller on QCS615.

Change-Id: I4e767e22bc9fc2cffabaa2562dfe7ac044cb3501
Signed-off-by: Li Liu <quic_lliu6@quicinc.com>
Signed-off-by: Fange Zhang <quic_fangez@quicinc.com>
Link: https://lore.kernel.org/all/20241113-add-display-support-for-qcs615-platform-v2-2-2873eb6fb869@quicinc.com/
Upstream-Status: Submitted [https://lore.kernel.org/all/20241113-add-display-support-for-qcs615-platform-v2-2-2873eb6fb869@quicinc.com/]
---
 .../devicetree/bindings/display/msm/dsi-controller-main.yaml     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/display/msm/dsi-controller-main.yaml b/Documentation/devicetree/bindings/display/msm/dsi-controller-main.yaml
index c6dbab65d5f7..32b2208abe05 100644
--- a/Documentation/devicetree/bindings/display/msm/dsi-controller-main.yaml
+++ b/Documentation/devicetree/bindings/display/msm/dsi-controller-main.yaml
@@ -22,6 +22,7 @@ properties:
               - qcom,msm8996-dsi-ctrl
               - qcom,msm8998-dsi-ctrl
               - qcom,qcm2290-dsi-ctrl
+              - qcom,qcs615-dsi-ctrl
               - qcom,sc7180-dsi-ctrl
               - qcom,sc7280-dsi-ctrl
               - qcom,sdm660-dsi-ctrl
-- 
2.34.1

